zuulcilint/__main__.py

```diff
@@ -177,15 +222,19 @@ def main():
 
     # Lint all Zuul YAML files
     results["errors"]["yaml"] = lint_all_yaml_files(zuul_good_yaml, schema)
-    results["warnings"]["file_extension"] = len(zuul_good_yaml)
+    results["warnings"]["file_extension"] = len(zuul_bad_yaml)
 
     # Check playbook paths if specified
     if args.check_playbook_paths:
         zuul_utils.print_bold("Checking playbook paths", "info")
         invalid_playbook_paths = lint_playbook_paths(zuul_good_yaml)
-        results["errors"]["playbook_paths"] = len(invalid_playbook_paths)
-        for path in invalid_playbook_paths:
-            print(f"  {path}")
+        if(invalid_playbook_paths):
+            results["errors"]["playbook_paths"] = len(invalid_playbook_paths)
+            zuul_utils.print_bold("  Invalid playbook paths:", "error")
+            for path in invalid_playbook_paths:
+                print(f"\t{path}")
+        else:
+            print("  No invalid playbook paths")
```

Comment:
Are the spaces at the start of the strings being printed there on purpose? Please use "\t" instead.

Reply:
Since there are other ocurrences in the existent code, I suggest we fix all of them in a followinf PR